doc/api/cli.md

```diff
@@ -1140,6 +1141,32 @@ directory provided as an argument. Coverage is output as an array of
 }
 ```
 
+Source Map data is output as an object, where the keys of the object represent
```

Comment:
IIUC, `NODE_V8_COVERAGE` will be the only way to generate sourcemaps with core initially?

Can we mark this feature as experimental?